components/ActionModals/TableInvoice.vue

```diff
@@ -82,10 +82,21 @@ export default {
       required: true,
     },
   },
-  data: () => ({
-    feeDenom: network.stakingDenom,
-  }),
   computed: {
+    feeDenom: {
+      get() {
+        // default to a denom if there is the only one being transacted and it has feeOptions specified
+        const transactionDenom =
+          this.amounts.length === 1 &&
+          this.fees.find(({ denom }) => denom === this.amounts[0].denom)
```

Comment:
do we need to put this term in "()" or does && bind enough?

Reply:
I used the linting and this stayed the same